src/Menge/MengeCore/SceneGraph/Context.cpp

```diff
@@ -40,7 +40,12 @@ Any questions or comments should be sent to the authors {menge,geom}@cs.unc.edu
 #include "GLScene.h"
 #include "graphCommon.h"
 #include "Select.h"
-#include "GL/glu.h"
+#ifdef __APPLE__
+#include <GLUT/glut.h>
```

Comment:
You can probably just include `OpenGL/glu.h` instead of glut (as it's not one of the original dependencies).